fatpack/pgcompact

```diff
@@ -4811,6 +4811,16 @@ $fatpacked{"PgToolkit/Registry/Compactor.pm"} = '#line '.(1+__LINE__).' "'.__FIL
   		user => $options->get(name => 'user'),
   		dbname => $arg_hash{'dbname'});
   
+  	my $default_session_params = {
+                          'lc_messages' => '\'C\'',
+                          'synchronous_commit' => 'off',
+                          'session_replication_role' => 'replica',
+                          'statement_timeout' => '\'0\''
+                          };
+  
+  	my $ops_session_params = $options->get(name => 'session-param');
+  	my $session_params = { %$default_session_params, %$ops_session_params };
```

Comment:
I believe default param hash should follow the custom one, not vice versa, as it shouldn't be allowed to redefine those 4 parameters that are in the default. And, BTW, it would probably be a good idea to rename $default_session_params to $hard_session_params.